.circleci/config.yml

```diff
@@ -623,7 +623,7 @@ jobs:
             echo "JSC_ENGINE = [os.path.expanduser('~/.jsvu/bin/spidermonkey')]" >> ~/emsdk/.emscripten
             echo "JS_ENGINES = [JSC_ENGINE]" >> ~/emsdk/.emscripten
```

Comment:
```suggestion
            echo "SPIDERMONKEY_ENGINE = [os.path.expanduser('~/.jsvu/bin/spidermonkey')]" >> ~/emsdk/.emscripten
            echo "SPIDERMONKEY_ENGINES = [JSC_ENGINE]" >> ~/emsdk/.emscripten
```

Reply:
(existing, but I noticed it now)

Reply:
Oh nice fine.. That is actually kind of key.